pkg/dump/pipeline/metadata.go

```diff
@@ -0,0 +1,28 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/DataDog/KubeHound/pkg/collector"
+	"github.com/DataDog/KubeHound/pkg/dump/writer"
+)
+
+type MetadataIngestor struct {
+	buffer map[string]*collector.Metadata
```

Comment:
Do we need that to be a pointer of `collector.Metadata` instead of simply `map[string]collector.Metadata` ?

Reply:
done